.github/workflows/master-workflow.yml

```
@@ -0,0 +1,17 @@
+name: Master Pipeline
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - master
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
```

This ended up being a problem with the Snyk scan. I'd suggest changing it like this:
```suggestion
on:
  push:
    branches:
      - master

jobs:
  publish_docs:
```

---

I'm confused, you mean this way?:
```
name: Master Pipeline
on:
  push:
    branches:
      - master
jobs:
  publish_docs:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    
    - name: Publish the docs
      run: yarn docs:publish
```

---

Yes.